src/Sensor.tsx

```diff
@@ -54,14 +54,16 @@ export const Sensor: React.FC<Props> = (props: Props) => {
     y: percToPx(sensor.position.y, imageDimensions.height),
   };
 
-  const mappingOperator = MappingOperators.find(mappingOperator => mapping?.operator === mappingOperator.id);
+  for(let mapping of mappings) {
+    const mappingOperator = MappingOperators.find(mappingOperator => mapping.operator === mappingOperator.id);
 
-  // Apply mapping function if it satisfies requirements
-  const isOverrode = mappingOperator?.function(value, mapping!.compareTo);
+    // Apply mapping function if it satisfies requirements
+    const isOverrode = mappingOperator?.function(value, mapping.compareTo);
 
-  if (isOverrode) {
-    // Assume that mapping values perfectly matches sensor fields, it's not covered by typescript
-    sensor = _.merge(sensor, mapping!.values);
+    if (isOverrode) {
```

Comment:
I'm not sure about this part. Is it OK if more than one mapping is overriding sensor values? Or maybe the for loop should be broken after the first one is handled?

Reply:
It's not the best for performance reasons but I think it's better for the user to keep it this way, I'm pretty sure it was like this in the older version too